app/models/account.rb

```diff
@@ -230,18 +230,17 @@ def fields_attributes=(attributes)
     fields     = []
     old_fields = self[:fields] || []
 
-    if attributes.is_a?(Hash)
-      attributes.each_value do |attr|
-        next if attr[:name].blank?
-
-        previous = old_fields.find { |item| item['value'] == attr[:value] }
-
-        if previous && previous['verified_at'].present?
-          attr[:verified_at] = previous['verified_at']
-        end
+    Array.wrap(attributes).each do |attr|
+      if !attr.is_a?(Hash) || attr[:name].blank?
+        raise ArgumentError, 'One of custom field attributes has invalid format'
```

Comment:
I'm sorry, I disagree with @nightpool on this... Raising ArgumentError will lead to error 500 in the API, so it won't be very useful. It's possible to raise ActiveRecord::RecordInvalid, but I am not 100% sure if this is the right place to do it (this is in a setter, not in the validation cycle...)

Until there is a better validation solution, silently ignoring seems better. Sorry for long delay.